src/activate/activate.c

```diff
@@ -180,7 +180,8 @@ static int exec_process(const char* name, char **argv, char **env, int start_fd,
         if (arg_inetd) {
                 assert(n_fds == 1);
 
-                r = rearrange_stdio(start_fd, start_fd, STDERR_FILENO); /* invalidates start_fd on success + error */
+                /* invalidates start_fd on success + error */
```

Comment:
Hmm, if the comment is moved to a separate line, it should be changed to active voice and capitalized:
`/* Invalidate start_fd ... */`

Reply:
OK, i can change that